### PR TITLE
	Update partner package versions

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @muratg \ @Eilon

File: test/Microsoft.AspNetCore.Mvc.Formatters.Xml.Test/XmlDataContractSerializerOutputFormatterTest.cs [hunk 1]
```diff
@@ -441,29 +438,6 @@ public void GetSupportedContentTypes_ReturnsSupportedTypes(Type type, object exp
             await Assert.ThrowsAsync(typeof(SerializationException),
                 async () => await formatter.WriteAsync(outputFormatterContext));
         }
-#else
-        // DataContractSerializer in CoreCLR does not throw if the declared type is different from the type being
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.Formatters.Xml.Test/XmlDataContractSerializerOutputFormatterTest.cs]: Ran this by @dougbu. They seem to have fixed the underlying issue so we no longer need the `#else`

File: test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs [hunk 2]
```diff
@@ -88,7 +88,7 @@ public ErrorPageTests(MvcTestFixture<ErrorPageMiddlewareWebSite.Startup> fixture
             Assert.Contains(PreserveCompilationContextMessage, content);
         }
 
-        [Fact]
+        [Fact(Skip = "Temporarily skipping to workound CoreCLR failure")]
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs]: Ongoing discussion in the email thread about this failure. I'll create a follow up work item to undo this

File: src/Microsoft.AspNetCore.Mvc.Abstractions/project.json [hunk 3]
```diff
@@ -37,16 +37,15 @@
...
     },
-    "Microsoft.Net.Http.Headers": "1.1.0-*"
+    "Microsoft.Net.Http.Headers": "1.1.0-*",
+    "NETStandard.Library": "1.6.1-*"
   },
   "frameworks": {
     "net451": {},
     "netstandard1.3": {
       "dependencies": {
         "Microsoft.CSharp": "4.0.1-*",
```
Comment (davidfowl) [inline src/Microsoft.AspNetCore.Mvc.Abstractions/project.json]: Is Microsoft.CSharp not in the standard library?

Comment (pranavkm) [inline src/Microsoft.AspNetCore.Mvc.Abstractions/project.json]: Err, missed this one. It's not in NETStandard.Library and should get bumped up to 4.3.0

**Comment by pranavkm** [comment]
@davidfowl - updated the missed Microsoft.CSharp package. The DiagnosticSources package is used in our tests which is where it's duplicated. 

🆙 📅

File: src/Microsoft.AspNetCore.Mvc.Abstractions/project.json (hunk 3 above)
Comment (davidfowl) [inline src/Microsoft.AspNetCore.Mvc.Abstractions/project.json]: hmm k

**Comment by davidfowl** [comment]
is the build supposed to be red?

**Comment by pranavkm** [comment]
Yeah. It's not using the dotnet-core feed. which is where the packages are. I'm working on kicking off the Mirror build so we pull their packages.

File: test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs (hunk 2 above)
Comment (muratg) [inline test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs]: Is this due to a product issue in CoreCLR?

Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs]: Yup - here's the issue for it https://github.com/dotnet/coreclr/issues/7381

**Pull request closed without merging**